services/statistics.py

```
@@ -0,0 +1,126 @@
+from datetime import datetime, timedelta
```

Ruff (0.8.0)

`datetime.datetime` imported but unused

Remove unused import

(F401)